Formula/j/jprq.rb

```diff
@@ -0,0 +1,17 @@
+class Jprq < Formula
+  desc "Join Public Router, Quickly"
+  homepage "https://jprq.io/"
+  url "https://github.com/azimjohn/jprq/archive/refs/tags/2.2.tar.gz"
+  sha256 "6121e0ac74512052ed00c57c363f0f0b66910618ebd8134cfa72acca05b09163"
+  license "BSD-3-Clause"
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cli"
+  end
+
+  test do
+    assert_match "auth token has been set", shell_output("#{bin}/jprq auth jprqbolmagin 2>&1")
+    shell_output("#{bin}/jprq serve .", 1)
```

Comment:
Can we check this?
```suggestion
    assert_match "something", shell_output("#{bin}/jprq serve .", 1)
```
Or should it be
```suggestion
    system bin/"jprq", "serve",  "."
```

Reply:
No, it does not work. serve command starts to serve folders, first it returns regular string like `Serving: 	 /private/tmp/jprq-test-20231022-33806-149a5qa` but after that authentication fails, and exception raised. Output thinks regular print is successful command output but it is not. 
<img width="836" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/74812737/e2b5c58b-33cd-4ed8-adc2-88d7964ba11c">

for first suggestion output 
<img width="651" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/74812737/6590d981-78f0-4c7f-ac4d-9fe6a5a7aa11">

I could not resolve that yesterday so i settled to just exit code is 1.

Reply:
@SMillerDev is there something else I could help with for this PR to be merged? Please let me know, thanks.